File: ruby/tutorial/comfy/16-collectdigits.rb
## -------------
## Tutorial 16 - collecting digits
## -------------
call = Tropo::TropoCall.new($incomingCall)
call.answer

result = call.ask 'Hello.  Please enter any number', :choices => '[DIGITS]'

if result.name == 'choice'
  call.say 'Great, you said ' + result.value
end
	
call.hangup

File: javascript/tutorial/16-collectdigits.js
// --------------------------------------------
// collect digits
// See http://www.tropo.com for more info
// --------------------------------------------

answer();

result = ask( "Hello.  Please enter any number", {choices:"[DIGITS]"} );

if (result.name=='choice') { say( "Great, you said " + result.value ) };
	
hangup();
